### ISSUE TITLE
Outreachy Documentation Project: Sukriti Sood

### ISSUE THREAD

**Issue by Sukriti-sood** [ersilia-os/ersilia#157]
Applicant: [Sukriti-sood](https://github.com/Sukriti-sood)
  
Welcome to the Ersilia Open Source Initiative. This issue will serve to track all your contributions for the project “Improve the documentation and outreach material of the Ersilia Model Hub”.
  
Please tick the tasks as you complete them. To make a final application it is *not* required to have completed all tasks. Only the *Initial Steps* and *Community* sections are REQUIRED. The tasks are not ordered from more to less important, they are simply related to different skills. Start where you feel most comfortable. This project can be adapted to the applicants interests, please focus on the type of tasks that you prefer / have better skills / would like to work on as an intern.

---
### Initial steps:
- [x] Record your application for the project in the outreachy website referencing this issue. Please make sure to select the right project on the website.
- [x] Join the Slack channel to follow public communications
- [x] Comment under this issue explaining why are you interested in this project
---
### GitHub documentation:
- [x] Create a README file with the name <README_yourname.md> under the /documentation folder
- [x] Link the #PR in a comment under this issue
- [x] Incorporate feedback from the mentor
---
### Writing dissemination material
- [x] Read the [Strategic plan 2021-2023](https://ersilia.gitbook.io/ersilia-wiki/) for Ersilia and create a 1-page blogpost with the main points
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [x] Incorporate feedback from the mentor
- [ ] Choose your own topic related to Ersilia (AI/ML for biomedical research, neglected diseases, drug discovery…) and write a 1-page blogpost to communicate to a non-expert audience
- [ ] Comment under this issue with a link to the blogpost (a google docs for example)
- [ ] Incorporate feedback from the mentor
- [ ] Create a template for a twitter post to release every time a new model is incorporated in the Hub (twitter: 280 characters, you can suggest a main post +  thread with extra information) and add it as a comment under this issue
- [ ] Create a template short Newsletter (1 paragraph) to send every month to our community (funders, users, contributors). It should mention metrics (models in the hub, number of users, funding…), thank you etc 
---
### Technical skills (required for the tutorial only)
- [x] [Install](https://ersilia.gitbook.io/ersilia-book/quick-start/installation) the Ersilia Model Hub
- [ ] Test one model
- [ ] Add a screenshot under this issue showing the model running in your computer
- [ ] Write a docstring for the ErsiliaModel class. Use the Google Python Style guide. Paste the docstring as a comment below (do not use a PR).
---
### Graphic material
- [ ] Read the [Ersilia Brand Guidelines](https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5)
- [ ] Read [“Why Ersilia?”](https://ersilia.gitbook.io/ersilia-wiki/)
- [ ] Create one image / slide to explain Ersilia’s mission and vision
- [ ] Link to the image/slide as a comment under this issue
- [ ] Incorporate feedback from the mentor
- [ ] Create two slides / short video showing how to use the Ersilia Model Hub and add them under the /tutorial folder 
- [ ] Link the #PR in this issue
- [ ] Incorporate feedback from the mentor
---
### Scientific content
- [x] Check the models available in the [Hub](https://airtable.com/shrXfZ8pqro0jjcsG/tblZGe2a2XeBxrEHP/viwd5XJVLslkE11Tg)
- [ ] Select one model from the list and write a technical card (what is the model for, what input, which data was used to create it, what kind of ML algorithm uses…) for it
- [ ] Add your card as a comment to this issue
- [ ] Search the scientific literature and suggest 3 new models (comment in this issue) that would be relevant to incorporate in the Hub.
---
### Community
- [x] Look up two other projects and comment on their issues with feedback on one of their tasks
- [x] If you have feedback from your peers, answer it in this issue.
---
### Other
If you have interest in working on related topics, or have new suggestions, please do the following
- [x] Add a comment in this issue with your new idea, tagging the mentor 
- [x] Get feedback from the mentor and act accordingly
- [x] Link in the comments any other PR you have contributed to.
---
### Final application
- [x] I have answered all comments from mentors and contributors
- [x] All PR or issues assigned to me are complete
- [x] I have submitted my final application to the project

**Comment by Sukriti-sood** [comment]
It has always been my desire to do something that contributes to improving Global Health. Ersilia is giving me the opportunity to participate in a team that is working on a related mission. I am an Open Source and Machine Learning enthusiast with a strong interest in Technical Writing, Backend Development. I love to work with communities as they foster collaborative learning. As a member of the poster-making group of my college club, I have experience with graphical tasks, and I have written various automation scripts in the past, so I have also become familiar with the Python programming language. The skills mentioned in the Projects are increasing my interest for the project.

**Comment by Sukriti-sood** [comment]
Create [PR](https://github.com/ersilia-os/ersilia/pull/167) for README file

**Comment by GemmaTuron** [comment]
Hi @Sukriti-sood thanks for your contribution!

Find here some feedback on the README file, think if there are any changes you would like to make based on it and continue working on other tasks.
I see you have done a lot of work on it, that's great! As a general comment, the README is quite long now and some information is repeated. I would suggest you think of each section and the main message you want to transmit in each section and then organize it accordingly
- In the About Us the Open Collective should have a link to our site to make it easier for people to follow
- Project description: the Ersilia Open Source Initiative is the organization, and the Ersilia Model Hub is one of our services, products, offers.. can you rephrase the first paragraph? The project description should simply be focused on the Ersilia Model Hub. The second paragraph should perhaps be related to a thank you or credit section.
- Overview: I very much like that schema, but I think it is perhaps too detailed for a README file
- Features: please revise the license
- Operations: I think you could eliminate this section and merge the content with the project description, as is very much related to the Ersilia Model Hub?
- License: please double check the License file in the repo
- The vision is part of the Ersilia Open Source Initiative, so perhaps it would be better placed with the About Us?
- Future Scope and Attributes: can you clarify what is the goal of these two sections? what main message you want to transmit?
- the Credit should mention how to cite our software, check the Citation File Format for more info on that

These are just some suggestions to guide your work, let me know what you think!

**Comment by GemmaTuron** [comment]
Hi @Sukriti-sood 
I am revising other contributors work and I realised some paragraphs are very similar or identical to contributions by other participants who worked on this issue before. Please make sure to post unique work, of course readme files will be similar, but we would appreciate if participants wrote the concepts in their own words.

**Comment by Sukriti-sood** [comment]
Thank you @GemmaTuron for your feedback, I will update the readme file according to your feedback. I had confusion regarding some of the points.. it got cleard by your feedback.

**Comment by Sukriti-sood** [comment]
@GemmaTuron I have updated Readme, according to the feedback .. please review it.

**Comment by Sukriti-sood** [comment]
As a result of reading the [Strategic Plan](https://ersilia.gitbook.io/ersilia-wiki/), I have created [this](https://docs.google.com/document/d/1t4zLkrHTlAfy2u2-G1_NdITn4J1WfpZqAYdJDWPTzWM/edit?usp=drivesdk) doc.

**Comment by GemmaTuron** [comment]
Hi @Sukriti-sood 

Thanks for the changes, I have merged your PR. I have also cloned your repo to work on the Organisation profile.
I think you should now focus on creating your final application! Thanks for all the work!

**Comment by Sukriti-sood** [comment]
> Hi @Sukriti-sood 
> 
> Thanks for the changes, I have merged your PR. I have also cloned your repo to work on the Organisation profile.
> I think you should now focus on creating your final application! Thanks for all the work!

Ook, Thanka✨

**Comment by GemmaTuron** [comment]
The Outreachy contribution period has come to an end, so I will close this issue. If you want to continue contributing to Ersilia, please follow the guidelines in the CONTRIBUTING file

**Comment by Sukriti-sood** [comment]
Thank you @GemmaTuron
